courses/management/commands/test_manage_program_certificates.py

```diff
@@ -0,0 +1,99 @@
+"""Tests for Program Certificates management command"""
+
+import pytest
+from courses.management.commands import manage_program_certificates
+from courses.models import ProgramCertificate
+from django.core.management.base import CommandError
+from courses.factories import (
+    CourseFactory,
+    CourseRunFactory,
+    CourseRunGradeFactory,
+    CourseRunCertificateFactory,
+    CourseRunEnrollmentFactory,
+    ProgramFactory,
+)
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_program_certificate_management_no_argument():
+    """Test that command throws error when no input is provided"""
+
+    with pytest.raises(CommandError) as command_error:
+        manage_program_certificates.Command().handle()
+    assert str(command_error.value) == "Please provide a valid program readable_id."
+
```

Comment:
You can add multiple tests along with this, e.g., When no argument (Like this) and then no program but a user was provided.